### PR TITLE
Update content blocks template include variables

### PR DESCRIPTION
**Pull request by tardinha**

### CODE REVIEW

File: themes/nswds/app/frontend/src/scss/components/waratah/_fixes.scss [hunk 1]
```diff
@@ -34,3 +34,17 @@
...
 }
+
+
+
+/**
+ * Add specific class for bitmap icons
+ */
+.nsw-content-block__image {
+  .nsw-content-block__icon {
+    img {
```
Comment (JamesDPC) [inline themes/nswds/app/frontend/src/scss/components/waratah/_fixes.scss:45]: Can this @extend https://github.com/digitalnsw/nsw-design-system/blob/master/src/components/content-block/_content-block.scss#L52 ?

File: themes/nswds/templates/nswds/Includes/ContentBlocks.ss
Comment (JamesDPC) [inline themes/nswds/templates/nswds/Includes/ContentBlocks.ss:1]: Would be more consistent if the changes were moved into the ElementalList ContentBlocks include, along the lines of Grid.ss and Cards.ss includes in the same location

File: themes/nswds/templates/nswds/Includes/ContentBlocks.ss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <% if $ContentBlocks_Items %>
-<div class="nsw-grid">
+<div class="nsw-grid nsw-grid--spaced">
```
Comment (JamesDPC) [inline themes/nswds/templates/nswds/Includes/ContentBlocks.ss:2]: Is spaced grid required ?

Comment (tardinha) [reply themes/nswds/templates/nswds/Includes/ContentBlocks.ss:2]: yep, [specifically for content blocks](https://digitalnsw.github.io/nsw-design-system/components/content-block/index.html).

**Review by JamesDPC — APPROVED** [review]

**Pull request merged**